### DISCUSSION TITLE
fa/manager/client-software-on-android/Tutorial-for-V2rayNG-app/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4572] [Content Help کمک به اصلاح محتوا]
# fa/manager/client-software-on-android/Tutorial-for-V2rayNG-app/

The proxy manager platform.

https://hiddify.com/fa/manager/client-software-on-android/Tutorial-for-V2rayNG-app/

**Comment by xeoncode** [comment]
با سلام. تو قسمت تست کانفیگ‌ها وقتی رو دکمه ای که گفتید میزنم خیلی طول میکشه تا اعداد نتیجه تست رو به میلی ثانیه بنویسه واسه کانفیگ ها. نمیشه این کارو سریعتر کرد ؟
